src/tiledimage.js

```diff
@@ -1186,6 +1226,22 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
             topLeft: topLeftTile,
             bottomRight: bottomRightTile,
         };
+    },
+
+    /**
+     * This function converts the top left viewport coordinates to pixels coordinates
+     * @param {Boolean} current Pass true for the current location; defaults to false (target location).
+     * @returns {OpenSeadragon.Point}
+     */
+    getOriginPixelCoordinate: function(current, withPixelDensityRatio) {
```

Comment:
We should think about whether we need this as part of the public API. My initial thought is that it should be private. 

Either way, it needs a better name, as "pixel" isn't specific enough. Maybe `getOriginDrawerCoordinate`.